docs/QuickStartGuide.md

```diff
@@ -0,0 +1,199 @@
+# TestGrid Quick Start Guide
+
+This provides the required steps in detail on how to run TestGrid.
+
+## Prerequisites
+
+1. Download testgrid distribution from [https://wso2.org/jenkins/job/testgrid/job/testgrid/lastSuccessfulBuild/artifact/distribution/target/WSO2-TestGrid.zip](https://wso2.org/jenkins/job/testgrid/job/testgrid/lastSuccessfulBuild/artifact/distribution/target/WSO2-TestGrid.zip)
+2. Extract **_WSO2-TestGrid.zip_** and copy [mysql connector](http://central.maven.org/maven2/mysql/mysql-connector-java/6.0.6/mysql-connector-java-6.0.6.jar) .jar file to `WSO2-TestGrid/lib` folder
+3. Set _TESTGRID_HOME_ environment variable to a desired folder
+4. Make sure _JAVA_HOME_ is set
+5. Create **_$TESTGRID_HOME/config.properties_**
+
+- _config.properties_ currently supports the following set of properties:
+   
+   ```properties
+   #database configurations
+   DB_URL
+   DB_USER
+   DB_USER_PASS
+   
+   #wum credentials
+   WUM_USERNAME
+   WUM_PASSWORD
+   
+   #AWS credentials and configurations
+   AWS_REGION_NAME
+   secretKey
+   accessKey
```

Comment:
I think user has to create a database. If the database is named 'testgriddb' under mysql with root/root user, then do not have to configure the above DB config. 

WUM is optional? 

AWS config should be set if the user is testing AWS testing via Cloudformation.